### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by pau-baiget**

### CODE REVIEW

**Comment by pau-baiget** [comment]
Internal tracking: b/142725102

File: .github/ISSUE_TEMPLATE/standard-bug-report-for-gapid.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: Standard Bug report for GAPID
+about: Create a report to help us improve
```
Comment (hevrard) [inline .github/ISSUE_TEMPLATE/standard-bug-report-for-gapid.md]: Create an actionable bug-report for GAPID

File: .github/ISSUE_TEMPLATE/standard-bug-report-for-gapid.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
```
Comment (hevrard) [inline .github/ISSUE_TEMPLATE/standard-bug-report-for-gapid.md]: most other topics are not verbs, but nouns, so maybe: `**Bug description**`

File: .github/ISSUE_TEMPLATE/standard-bug-report-for-gapid.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (hevrard) [inline .github/ISSUE_TEMPLATE/standard-bug-report-for-gapid.md]: Turn into a noun: `Reproduction steps`

File: .github/ISSUE_TEMPLATE/standard-bug-report-for-gapid.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment information:**
+ - GAPID version:
+ - Host OS: [e.g. Linux]
+ If tracing on Android:
+ - Device model:
+ - Android Version:
```
Comment (hevrard) [inline .github/ISSUE_TEMPLATE/standard-bug-report-for-gapid.md]: I would put this one as the first item.

File: .github/ISSUE_TEMPLATE/standard-bug-report-for-gapid.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+** Additional debugging information**
```
Comment (hevrard) [inline .github/ISSUE_TEMPLATE/standard-bug-report-for-gapid.md]: Remove space before "Additional"

**Pull request merged**